docs/rules/valid-local-dependency.md

```diff
@@ -0,0 +1,25 @@
+# Ensures that local dependencies specified in a package.json exist
+
+## Rule Details
+
+This rule validates the casing for `file:` and `link:` dependencies in a `package.json` file. If the casing of a `link:` or `file:` path is incorrect yarn does not sub out the link when releasing.
+
+Examples of **incorrect** code for this rule:
```

Comment:
[Docs] I know this doesn't quite adhere to the normal rule docs convention, but it'd be good to mention what the right path is... maybe something like this?

```suggestion
Examples of **incorrect** code for this rule (when `../folder` is the correct path):
```

(and similar for the correct code)